src/content/docs/browser/new-relic-browser/configuration/proxy-agent-requests.mdx

```diff
@@ -0,0 +1,75 @@
+</Callout>
+
+## How to use a proxy [#use]
+
+The agent utilizes two configuration properties when determining where to fetch agent code or send observability data.
+
+For example, using copy-paste agent:
+
+```js
+window.NREUM.init = {
+    ...<other init properties>,
+    proxy: {
+        assets: <URL to fetch agent chunks>,
+        beacon: <URL to send analytics>
+    }
+}
+```
+
+For NPM, similarly just pass the `init` object with a proxy as part of any other configurations:
+
+```js
+new BrowserAgent({ init: { proxy: <object similar to above> } })
+```
+
+`init.proxy.assets` is used verbatim in the copy-paste installation to fetch agent code.
+
+* For example, the v1.240.0 Pro+SPA loader will require `nr-spa.<hash>-1.240.min.js` file.
+* If this field is set to `nrassets.abc.com``, the agent code will be loaded from `https://nrassets.abc.com/nr-spa.<hash>-1.240.0.min.js` instead of `https://js-agent.newrelic.com/nr-spa.<hash>-1.240.0.min.js`.
+
+This configuration is not relevant to NPM installed agents. NPM installed agents will undergo the same bundling process as the host application and should result in agent assets in the output directory along side the host application. When the host application is deployed, those agent assets should be deployed as well resulting in agent code already loading from the same domain as the site.
+
+`init.proxy.beacon` is used by both installation methods to allow proxying of observability data through a different domain. For example, if this field is set to `nrbeacon.abc.com`, all observability data will be sent to `https://nrbeacon.abc.com/<observability endpoint>` instead of what is configured automatically on the New Relic account and found in the `NREUM.info.beacon` configuration.
+
+<Callout variant="important">
+These configurations should follow a pattern of `<domain>[:<port>][/<path>]`. The protocol `https://` and ending slash **should be omitted**. The `port` and `path` are optional and will be respected.
+</Callout>
+
+
+<Callout variant="tip">
+The browser agent team recommends using different values for each proxy configuration. This may mean each configuration is a different subdomain or possibly each contains a different path. This will make configuring the proxy server a lot easier.
+</Callout>
+
+## About your proxy server [#expectations]
+
+The New Relic browser agent team provides neither a cloud proxy service nor support on such service application. **You must make and maintain your own proxy servers.**
+
+To that effect, some assumptions are made about your proxy that must hold true for full functionality:
+
+1. As mentioned above, if a proxy is used to load the agent code, it should fetch them from our CDN and pass it back to the client.
```

Comment:
```suggestion
* If a proxy is used to load the agent code, the proxy should fetch them from our CDN and pass it back to the client.
```
@cwli24 What's `them` in this context? The agent code?

Reply:
oh, the `them` refers to the agent code (parts)